### PR TITLE
added uwv-scholingsvoucher

### PR DESCRIPTION
**Pull request by tanjahennis**
I have added the UWV scholingsvoucher to the homepage :) Next step is turning the button into an actual button.  

screen shot 2016-10-27 at 10 32 17

### CODE REVIEW

File: app/views/static_pages/home.html.slim [hunk 1]
```diff
@@ -29,3 +29,16 @@
...
+    .row
+      .col.s12.m4
+        .row
+          = image_tag ('illustrations/uwv-voucher.png')
+      .col.s12.m8
+        .row
+          h2 UWV Scholingsvoucher
+          p The Dutch Government wants you... to become a web developer! Here at Codaisseur we specialize in junior web-developer training. Once you have passed our selection process we get you job-ready in a two-month intensive training program.
+        .row
+          p button
```
Comment (miriamtocino) [inline app/views/static_pages/home.html.slim]: Why is the proper button not there yet?
I guess @durw4rd still need to add it to the `shared/` folder :)

File: app/views/static_pages/home.html.slim [hunk 2]
```diff
@@ -29,3 +29,43 @@
...
+          .col.s12.m4
+            .card.horizontal
+              .card-image
+                = image_tag ('illustrations/rocket_black.png')
+              .card-stacked
+                .card-content
+                  h4 Open Evening
+                  p Come and ask all your questions
+                .rsvp
+                  .waves-effect.waves-light.btn RSVP
```
Comment (miriamtocino) [inline app/views/static_pages/home.html.slim]: Extra line at the end of the file :wink:

File: app/views/static_pages/home.html.slim (hunk 1 above)
Comment (tanjahennis) [inline app/views/static_pages/home.html.slim]: yes - I made this commit yesterday evening and hadn't pulled the newest version yet

File: app/views/static_pages/home.html.slim (hunk 2 above)
Comment (tanjahennis) [inline app/views/static_pages/home.html.slim]: I'm working on it ;) I'll three-double check with my next commit

**Pull request merged**